lib/routers/establishment/projects.js

```diff
@@ -171,7 +199,7 @@ router.param('projectId', (req, res, next, projectId) => {
             .orWhere('additionalEstablishments.id', req.establishment.id);
         })
         .withGraphFetched(
-          '[licenceHolder(constrainParams), collaborators(constrainParams).establishments, establishment(constrainEstablishmentParams), additionalEstablishments(constrainAdditionalEstablishmentParams)]'
+          '[licenceHolder(constrainParams), collaborators(constrainParams).establishments, establishment(constrainEstablishmentParams), additionalEstablishments(constrainAdditionalEstablishmentParams), retrospectiveAssessments]'
```

Comment:
This makes me sad.

Does it work with backticks and newlines?

```
`
[
  licenceHolder(constrainParams),
  collaborators(constrainParams).establishments,
  establishment(constrainEstablishmentParams),
  additionalEstablishments(constrainAdditionalEstablishmentParams),
  retrospectiveAssessments
]
`
```

Reply:
yeah I know what you mean, I really have no idea why it has to be a string array, and not an array of strings. Backticks should work ok though